bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -92,6 +91,9 @@ class BusinessPartnerService(
 
         saveChangelog(resolutionResults)
 
+        val partners = resolutionResults.map { it.businessPartner }
+        sharingStateService.setInitial(partners.map { it.externalId }, BusinessPartnerType.ADDRESS)
```

Comment:
I added it via a simple logic to map from business partner types to the relevant BPN.

Later on, we should move from setting BPN and business partner type explicitely in the sharing state and rather inferring it from the business partner stages. But this can only happen after we integrated the L/S/A endpoints to the generic model, made the external identifiers unique and the business partner type optional in the sharing state.